spec/folio_sync/archives_space_to_folio/folio_synchronizer_spec.rb

```diff
@@ -191,4 +191,48 @@
       )
     end
   end
-end
+
+  describe '#update_archivesspace_records' do
+    let(:pending_records) do
+      [
+        instance_double(
+          AspaceToFolioRecord,
+          repository_key: 1,
+          resource_key: 123,
+          folio_hrid: 'hrid1',
+        ),
+        instance_double(
+          AspaceToFolioRecord,
+          repository_key: 2,
+          resource_key: 456,
+          folio_hrid: 'hrid2',
+        )
+      ]
+    end
```

Comment:
This works, but there's a great gem called `factory_bot` that's useful for easily generating instances of ActiveRecord objects (and other objects generally) for test purposes:

https://github.com/thoughtbot/factory_bot

And for use with Rails, the `factory_bot_rails` gem is a good way to include the factory_bot gem: https://github.com/thoughtbot/factory_bot_rails

Reply:
Confirming this will be done as a follow-up PR